solarkraft/.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 dist
+_apalache-out
+entry-0xasdf.json
```

Comment:
Why is this here? Looks like a leftover?

Reply:
If you look at `asdf/100/entry-timelock.json` test file, even though it's called that, the contents _inside_ the file say
```
"txHash": "0xasdf",
"contractId": "0xqwer",
```
instead of
```
"txHash": "timelock",
"contractId": "asdf",
```
So if you actually run `verify` on this file, the output will be saved to a file, the name of which will be synthesized from the content, i.e. `0xqwer/100/entry-0xasdf.json`

Although I suppose if the gitignore edit bothers you, an easier soloution would probably be to change the hash and id in the original json.

Reply:
> Although I suppose if the gitignore edit bothers you

It's not about what bothers me.
Renaming a misnamed file just looks like the more immediate solution.